app/client/cypress/e2e/Regression/ClientSide/AdminSettings/Email_settings_Spec.ts

```
@@ -0,0 +1,507 @@
+            console.log("Email:", email);
+            expect(email).to.exist;
+            expect(email.headers.subject).to.include(inviteEmailSubject);
+            expect(email.headers.to).to.include(emailThree);
+            const emailHtml = email.html; // Store the email HTML content
+
+            // Match all href links inside the <body>
+            const bodyMatch = emailHtml.match(/<body[^>]*>([\s\S]*?)<\/body>/i);
+            console.log("bodyMatch: ", bodyMatch);
+
+            if (bodyMatch && bodyMatch[1]) {
+              const bodyContent = bodyMatch[1];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)